.github/workflows/helm-docs.yaml

```
@@ -0,0 +1,38 @@
+
+name: "Generate Helm Docs"
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  helm-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)